src/api/app/views/source/projects.xml.builder

```diff
@@ -0,0 +1,4 @@
+xml.directory() do
+  @projects.map { |project| xml.entry(name: project) }
```

Comment:
Just copied from the "view" in the controller, each makes probably more sense, yes!